templates/xlang/experimental.yaml

```diff
@@ -45,7 +46,8 @@ spec:
         app: xlang-{{ $langserver.language }}
     spec:
       containers:
-      - image: sourcegraph/codeintel-{{ default $langserver.language (index (dict "cs" "csharp" "dockerfile" "docker") $langserver.language) }}
+      {{ $imageName := (pluck "image" (pluck $langserver.language $experimentalLangServers | first) | first) -}}
+      - image: {{ $imageName }}
```

Comment:
I'm not 100% sure about the syntax, but it looks like this is just grabbing a nested field. Could you use [`index`](https://godoc.org/text/template)? Something like this:

```
- image: {{ index $experimentalLangServers $langserver.language image }}
```